pkgs/tools/misc/yubikey-manager-qt/remove-upload-import.diff

```diff
@@ -0,0 +1,15 @@
+--- a/ykman-gui/py/yubikey.py	2025-01-08 19:42:20.058393985 +0100
++++ b/ykman-gui/py/yubikey.py	2025-01-08 19:44:12.520364702 +0100
+@@ -50,12 +50,8 @@
+ if int(ykman_v.split(".")[0] ) > 4:
+     from yubikit.support import get_name
+     from ykman.device import list_all_devices, scan_devices
+-    from ykman.otp import (
+-    _PrepareUploadFailed as PrepareUploadFailed
+-    , _prepare_upload_key as prepare_upload_key, generate_static_pw)
```

Comment:
`generate_static_pw` still exists it seems, and the import line below is valid since it's under a version check.

Reply:
Thank you, fixed.